pyproject.toml

```diff
@@ -41,6 +35,7 @@ dependencies = [
     "click-plugins>=1.0",
     "cligj>=0.5",
     'importlib-metadata;python_version<"3.10"',
+    "numpy>=1.25,<2",
```

Comment:
It's not _needed_ to require a numpy 1.25 as a minimum version. The goal of _building_ with a minimum version of 1.25 is that this ensures it also runs with older numpys  
(in contrast to older versions of numpy, where building with that specific version meant that the wheels were only compatible with that version or up, hence the use of "oldest-supported-numpy" for wheel building)

See https://numpy.org/doc/stable/release/1.25.0-notes.html#compiling-against-the-numpy-c-api-is-now-backwards-compatible-by-default in the 1.25 release notes, and https://numpy.org/doc/stable/dev/depending_on_numpy.html#build-time-dependency